README.md

```diff
@@ -28,59 +28,19 @@ Clio Identity is Clio's authentication service and identity provider. Similar to
 
 To request a Clio Identity API key in order to implement SSO see our guide for [integrating with Clio Identity](https://developers.support.clio.com/hc/en-us/articles/4405288237723-Integrating-with-Clio-Identity-Single-Sign-on-with-Clio-).
 
-### PostgreSQL
+### Setup
 
-This application utilizes [PostgreSQL](https://www.postgresql.org/) as it's database system. 
+First, clone the example application repository to your local environment: `git clone https://github.com/clio/example-third-party-application.git`.
 
-For macOS users, we recommend using [Postgres.app](https://postgresapp.com/), which is a simple, native macOS app that runs in the menubar without the need of an installer. Open the app, and you have a PostgreSQL server ready and awaiting new connections. Close the app, and the server shuts down.
-
-On macOS installing the `postgresql` package is also required to be able to use the `psql` client command, as well as to install any libraries that are required to build the `pg` driver for use with `rails`.
-
-```
-brew install postgresql
-```
-
-### Ruby and Rails
-
-This application utilizes Ruby 2.6.8 and Rails 5.2.6. These can be installed through a variety of methods using different package managers, choose the right one for your system and preferences.  
-
-## Installation
-
-To get the example third-party application running on your local environment, ensure that you have valid API key pairs for both Clio Manage and Clio Identity, and that you have PostgreSQL, Ruby and Rails installed, as described in the Requirements section above.
-
-Clone the repository to your local environment:
-
-```
-git clone https://github.com/clio/example-third-party-application.git
-```
-
-Update the environment variables in `config/local_env.yml`:
+Then update the environment variables in `.env`:
```

Comment:
I had to change `local_env.yml` for the app to load the environment variables properly. It doesn't seem to be loading from the `.env` instead.  I may have missed a step?

Are you intending to use `dotenv-rails` and `Dotenv.load` in the config?